### PR TITLE
fix: open MT/CT Editor in Keybord mode when using iOS

### PR DESCRIPTION
**Pull request by icaparros-at-wiris**
## Description

When the user opens the MT/CT Editor using an iOS device with zoomed webpage. The user will not be able to zoom out on the handwrite area.

Now, the MT/CT Editor will be open on keyboard mode when using an iOS device by default.

If the user use Safari and doesn't zoom, the page will be opened in handwrite mode. This will not happen in Chrome, as I didn't found a way to detect the zoom level of the page.

## Steps to reproduce

1. Open a demo on `html-integrations`
2. With an iOS device, navigate to the opened demo using Safari
3. Zoom the web page
4. Open the MT Editor

The MT Editor will be opened in Keyboard mode

5. Close the MT Editor
6. Reset the page
7. Open the MT Editor without zooming the page

The MT Editor will be opened in Handwrite mode

---

[#taskid 24192](https://wiris.kanbanize.com/ctrl_board/2/cards/24192/details/)

### CODE REVIEW

File: packages/mathtype-html-integration-devkit/src/contentmanager.js [hunk 1]
```diff
@@ -447,6 +479,33 @@ export default class ContentManager {
     }
     this.updateToolbar();
     this.onFocus();
+
+    if (this.deviceProperties.isIOS) {
+      if (this.browserProperties.isSafari) {
+        const zoom = document.documentElement.clientWidth / window.innerWidth;
```
Comment (xripoll-at-wiris) [inline packages/mathtype-html-integration-devkit/src/contentmanager.js:485]: Does this not work with browsers other than Safari? What's the behavior there?

File: packages/mathtype-html-integration-devkit/src/contentmanager.js [hunk 2]
```diff
@@ -16,14 +16,15 @@ export default class ContentManager {
    * - The communication between the {@link ModalDialog} class and itself, if the content
    *   has been changed (hasChanges} method).
    * @constructs
-   * @param {Object} contentManagerAttributes - Object containing all attributes needed to
+   * @param {object} contentManagerAttributes - Object containing all attributes needed to
```
Comment (xripoll-at-wiris) [inline packages/mathtype-html-integration-devkit/src/contentmanager.js:19]: There is a lot of linting changes; maybe those shouldn't be part of the PR since there's so many. In any case we can have another task/PR linting the devkit source files.

File: packages/mathtype-html-integration-devkit/src/contentmanager.js (hunk 1 above)
Comment (icaparros-at-wiris) [reply packages/mathtype-html-integration-devkit/src/contentmanager.js]: I checked that using an iPhone, and it is true. I removed the isSafari condition.

**Review by carla-at-wiris — APPROVED** [review]

**Review by carla-at-wiris — APPROVED** [review]
